index.html

```diff
@@ -0,0 +1,532 @@
+                    <img src="img/works/Img-placeholder.svg" alt="">
+                </div>
+                <div id="works-right-block">
+                    <h3 id="works-title-post">Multi-Post Stories</h3>
+                    
+                    <p id="works-supporting-text">
+                        A daily selection of privately 
+                        personalized<br> reads; no accounts 
+                        or sign-ups required.<br> has been 
+                        the industry's standard dummy<br> 
+                        text ever since the 1500s, 
+                        when an<br> unknown printer took 
+                        a standard dummy<br> text.
+                    </p>
+
+                    <div id="works-tags">
+                        <p class="tag">css</p>
+                        <p class="tag">html</p>
+                        <p class="tag">bootstrap</p>
+                        <p class="tag">Ruby </p>
+                    </div>
+
+                    <button id="works-enable-button"><p class="see-project-text">See Project</p></button>
+                </div>
+            </div>
+
+            <div id="works-frame43">
+                <div class="frame43-card-works1 card-work">
+                    <div class="frame43-card-works-mask-group">
+                        <div class="card-works-img1"></div>
+                    </div>
+                    
+                    <div class="frame43-card-works-right-block">
+                        <h3 class="frame43-card-works-right-block-title-post"><p id="works-title-text">Profesional Art Printing Data</p></h3>
+
+                        <p class="frame43-card-works-right-block-supporting-text">
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required. has been the industry's standard
+                        </p>
+
+                        <div class="frame43-card-works-right-block-tags">
+                            <div class="frame43-card-works-right-block-html-tag">
+                                <div class="frame43-card-works-right-block-tag">
+                                    <p class="works-tag-text">html</p>
+                                </div>
+                            </div>
+    
+                            <div class="frame43-card-works-right-block-bootstrap-tag">
+                                <div class="frame43-card-works-right-block-tag">
+                                    <p class="works-tag-text">bootstrap</p>
+                                </div>
+                            </div>
+    
+                            <div class="frame43-card-works-right-block-ruby-tag">
+                                <div class="frame43-card-works-right-block-tag">
+                                    <p class="works-tag-text">Ruby</p>
+                                </div>
+                            </div>
+                        </div>
```

Comment:
- [ ] - [ ] Kindly ensure that the list of project technologies (e.g., "JavasScript", "Ruby" and "CSS") are a list `ul` and each of them a `li` which is semantic as `<ul>` is used to represent a list 👍🏽.